Godeps/_workspace/src/k8s.io/kubernetes/pkg/proxy/hybrid/proxier.go

```diff
@@ -0,0 +1,130 @@
+package hybrid
```

Comment:
We should come up with a clever name that means both idling and unidling, and then use "unidle" and "idle" as our package names ("ing" is untidy).  We can worry about that right at the end though, but people should suggest good names that cover both parts.